src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Toolbar.js

```diff
@@ -11,7 +14,41 @@ export default class Toolbar extends React.PureComponent {
         return (
             <header className={toolbarStyles.toolbar}>
                 <nav>
-                    {toolbarStore.items.map((item) => (<Item key={item.title} {...item} />))}
+                    <div className={toolbarStyles.pullLeft}>
```

Comment:
It would make more sense if the pull-left class was global. Then you could compose classes/styling. e.g. 
class="toolbar-items pull-right"